tests/components/homekit_controller/common.py

```diff
@@ -101,6 +102,41 @@ def __init__(self, hass, entity_id, pairing, accessory):
         return state
 
 
+def create_generic_service(service_name):
+    """Create a test HomeKit service model."""
+    from homekit.model.services import AbstractService, ServicesTypes
```

Comment:
In tests we can import requirements at the top of the module if the requirements have been added to `requirements_test_all.txt`.

Reply:
Oh that’s good to know, that makes this a lot nicer.